src/libstd/primitive_docs.rs

```diff
@@ -10,8 +10,46 @@
 
 #[doc(primitive = "bool")]
 //
-/// The boolean type.
+/// The boolean type represents a binary value, which could either be `true` or `false`.
```

Comment:
It is easier to understand this explanation if the word "binary" is skipped here, it has so many meanings.